src/lib/helper.js

```diff
@@ -0,0 +1,18 @@
+const filter = (name) => {
+  name = name.toLowerCase().trim();
+  let pattern = /[.,:'?!;\-_)({}[\]¡¿"—*%#^]*/g;
+  return name.replace(pattern, '');
+};
```

Comment:
Here comes the fun part. 😅 Love that you used regex. 
I'm really bad at regex but I think there are a few shorter ways to check for something, like `\w` for letters, `\W` for special characters, or `\d` for digits. I think those all I can remember right now. 😅

Reply:
This [w3school link] (https://www.w3schools.com/jsref/jsref_obj_regexp.asp) is about regex and they are mentioning the characters I was talking about. There are lots of them. 😅

Reply:
Regex has never been a strength for me. I imagine it did not start out easy for you either when you attempted this issue. Great work figuring it out💪

Reply:
> Here comes the fun part. 😅 Love that you used regex.
> I'm really bad at regex but I think there are a few shorter ways to check for something, like `\w` for letters, `\W` for special characters, or `\d` for digits. I think those all I can remember right now. 😅

We would love to use these metacharacters and delete all this nonsense but doing so will restrict the items to be only in English